benchmarking/frameworks/caffe2/caffe2.py

```diff
@@ -256,6 +256,10 @@ def composeRunCommand(self, platform, program, model, test, model_files,
         cmd = [str(s) for s in cmd]
         return cmd
 
+    def runPreprocessOnPlatform(self, cmd, platform):
+        # run the preprocess command
+        platform.runBenchmark(cmd)
```

Comment:
for now, can we remove this function and call it directly in framework_base.py? Do you think we will have different implementations for different frameworks? Or most likely they are the same? If it is the latter, maybe we can worry about the different implementations when we see them.

Reply:
agree!

Reply:
This is not deleted yet.